docs/conceptual/array.comparewith['t]-function-[fsharp].md

```diff
@@ -0,0 +1,81 @@
+---
+title: Array.compareWith<'T> Function (F#)
+description: Array.compareWith<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+manager: danielfe
+ms.date: 07/07/2016
+ms.topic: language-reference
+ms.prod: visual-studio-dev14
+ms.technology: devlang-fsharp
+ms.assetid: 39df9501-99de-4ed3-b1d6-3aab44135861
+---
+
+# Array.compareWith<'T> Function (F#)
+
+Compares two arrays using the given comparison function, element by element.
+
+**Namespace/Module Path**: Microsoft.FSharp.Collections.Array
+
+**Assembly**: FSharp.Core (in FSharp.Core.dll)
+
+## Syntax
+
+```fsharp
+// Signature:
+Array.compareWith : ('T -> 'T -> int) -> 'T [] -> 'T [] -> int
+
+// Usage:
+Array.compareWith comparer source1 source2
+```
+
+#### Parameters
+*comparer*
+Type: **'T -&gt; 'T -&gt;**[int](https://msdn.microsoft.com/library/025d5455-3622-4ea5-9573-3ecbd4ee1375)
+
+A function that takes an element from each of the two source arrays and returns an int. If it evaluates to a non-zero value, iteration is stopped and that value is returned.
+
+*source1*
+Type: **'T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
+
+The first input array.
+
+*source2*
+Type: **'T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
+
+The second input array.
+
+## Return Value
+Returns the first non-zero result from the comparison function. If the first array has a larger element, the return value is always positive. 
+If the second array has a larger element, the return value is always negative. 
```

Comment:
@cartermp I had it there because it was getting hard to read that long line in my editor and because won't appear in the final display.